src/OrleansBenchmarks/project.json

```diff
@@ -0,0 +1,25 @@
+﻿{
+  "dependencies": {
+    "BenchmarkDotNet": "0.9.0",
+    "Microsoft.CodeAnalysis.Analyzers": "1.1.0",
+    "Microsoft.CodeAnalysis.Common": "1.0.0",
+    "Microsoft.CodeAnalysis.CSharp": "1.0.0",
+    "Microsoft.Diagnostics.Tracing.TraceEvent": "1.0.41",
+    "System.Collections": "4.0.0",
+    "System.Diagnostics.Debug": "4.0.0",
+    "System.Globalization": "4.0.0",
+    "System.Linq": "4.0.0",
+    "System.Resources.ResourceManager": "4.0.0",
+    "System.Runtime": "4.0.0",
+    "System.Runtime.Extensions": "4.0.0",
+    "System.Threading": "4.0.0",
+    "System.Threading.Tasks": "4.0.0"
```

Comment:
we are now using nuget transitive restore, so please just reference the nuget packages you depend on, but not the dependencies of your direct dependencies. This ways upgrades will be easier for us (and also migration to .net core)